Adding Diff command (#217)
FairwindsOps/reckoner: docs/README.md

  --help            Show this message and exit.

Commands:
  plot           Install charts with given arguments as listed in yaml file argument
  template       Output the template of the chart or charts as they would be installed or
  upgraded
  get-manifests  Output the manifests of the chart or charts as they are installed
  diff           Output diff of the templates that would be installed and the manifests as they are installed

>   update         Checks to see if any thing will have changed, if so, updated the release, if not, does nothing

Reviewer: ```suggestion
  update         Checks to see if anything will have changed, if so, update the release, if not, does nothing
```

  --help                          Show this message and exit.
```

Or
```
# reckoner update --help
Usage: reckoner update [OPTIONS] COURSE_FILE

>   Checks to see if any thing will have changed, if so, updated the release,

Reviewer: ```suggestion
  Checks to see if anything will be changed, if so, update the release,
```
